Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -77,11 +74,19 @@ namespace gdjs {
 
     ensureUpToDate() {
       if (this._spritesContainer.visible && this._wasRendered) {
-        // Cache the rendered sprites as a bitmap to speed up rendering when
-        // lots of panel sprites are on the scene.
-        // Sadly, because of this, we need a wrapper container to workaround
-        // a PixiJS issue with alpha (see updateOpacity).
-        this._spritesContainer.cacheAsBitmap = true;
+        // PIXI uses PIXI.SCALE_MODES.LINEAR for the cached image:
+        // this._spritesContainer._cacheData.sprite._texture.baseTexture.scaleMode
+        // There seems to be no way to configure this so the optimization is disabled.
```

Comment:
Should we create an issue on their repository and link the issue in the comment here? 
This would help with documentation when we come back to it one day